corehq/motech/dhis2/entities_helpers.py

```diff
@@ -27,6 +18,34 @@
     get_case_trigger_info_for_case,
     get_value,
 )
+from dataclasses import dataclass
```

Comment:
You'll be wanting to install [isort](https://pycqa.github.io/isort/). The codebase includes configuration for it. I have it mapped to Alt-I in my IDE to make it quick and easy. It will keep import order consistent, clean, and avoid unnecessary merge conflicts.